cms/core/migrations/0007_remove_glossaryterm_core_glossary_term_name_unique_and_more.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 5.1.7 on 2025-04-09 14:41
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def add_new_locale(apps, schema_editor):
+    Locale = apps.get_model("wagtailcore", "Locale")
+    Locale.objects.update_or_create(language_code="cy")
+
+
+def remove_new_locale(apps, schema_editor):
+    Locale = apps.get_model("wagtailcore", "Locale")
+    Locale.objects.filter(language_code="cy").delete()
```

Comment:
Note: We don't have a reverse migration strategy for this. The user should decide what to do with translated pages before running the migration.